src/server/database/migrations/20171026160540-removeBillingFromPersonalTeams.js

```diff
@@ -0,0 +1,20 @@
+exports.up = (r) => {
+  const billingFields = [
+    'stripeId',
+    'stripeSubscriptionId',
+    'periodEnd',
+    'periodStart'
+  ];
+  return r
+    .table('Organization')
+    .filter({tier: 'personal'})
+    .replace((org) =>
+      org
+        .without(...billingFields)
+        .merge({creditCard: {}})
```

Comment:
I did this based on what I was seeing in the system already. That said, intuitively it feels like this field shouldn't exist on teams that aren't (or aren't planning to) pay for a pro account yet. Anyways, I'm curious as to your thoughts.

Reply:
Yeah, removing it should be OK, but we should search for `creditCard` to make sure it supports a null whenever we call it.